### PR TITLE
[docs] Remove multiple metrics reference in scoring argument for TuneSearchCV

### PR DESCRIPTION
**Pull request by sumanthratna**
This change only affects the docstring for the `scoring` keyword argument in `TuneSearchCV`'s initializer. 

See #53 (note: this PR makes no attempt to implement multiple metrics)

CC @inventormc and @richardliaw

### CODE REVIEW

File: tune_sklearn/tune_search.py [hunk 1]
```diff
@@ -80,15 +80,10 @@ class TuneSearchCV(TuneBaseSearchCV):
...
-            or None): A single string (see Scikit-Learn documentation
-            on `scoring_parameter`) or a callable
+        scoring (str, `callable`, or None): A single string (see 
+            Scikit-Learn documentation on `scoring_parameter`) or a callable
             to evaluate the predictions on the test set.
-            For evaluating multiple metrics, either give a list of (unique)
-            strings or a dict with names as keys and callables as values.
-            NOTE that when using custom scorers, each scorer should return a
-            single value. Metric functions returning a list/array of values
-            can be wrapped into multiple scorers that return one value each.
```
Comment (sumanthratna) [inline tune_sklearn/tune_search.py:91]: I removed, "Metric functions returning a list/array of values can be wrapped into multiple scorers that return one value each." but I'm not entirely sure if this actually hasn't been implemented in tune-sklearn.

Comment (inventormc) [reply tune_sklearn/tune_search.py:91]: Yes, we haven't implemented this yet. Thanks!

**Comment by anthonyhsyu** [comment]
Thank you for the catch and PR @sumanthratna!
